### ISSUE TITLE
What happened to this game?

### ISSUE THREAD

**Issue by AntonioS3** [Menencia/clickingff7#14]
Well, this game is dead. It have no updates... what's up? It's been 9 months like the last update.

**Comment by Equinox** [comment]
Actually, @Menencia seems extremely busy and has a working branch that you can find [here](https://github.com/Menencia/clickingff7/tree/feature/next2) that has a commit less than 2 weeks ago :)

**Comment by Menencia** [comment]
Yep, I'm working on the next version. More infos [here](https://www.reddit.com/r/clickingff7/comments/3bqkgo/news_0107/) and soon.
